wiki/osu!_Program_Files/User_configuration_file/en.md

```diff
@@ -1,230 +1,232 @@
-| MenuMusic                    | Menu start-up music toggle                                                                                                                                                                                 |
-| MenuVoice                    | menu Voices toggle                                                                                                                                                                                         |
-| AutomaticCursorSizing        | Toggle automatic cursor sizing to map's circle size                                                                                                                                                        |
-| ShowInterface                | interface hiding controlled by shift+tab default                                                                                                                                                           |
-| @AutoSendNowPlaying          |                                                                                                                                                                                                            |
-| AutoSendNowPlaying           | Decides whether player auto /np's the map to spectator chat                                                                                                                                                |
-| keyDecreaseSpeed             |                                                                                                                                                                                                            |
-| UsePerBeatmapManiaSpeed      |                                                                                                                                                                                                            |
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
-||
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must.
+
```

Comment:
To the table: Add a note that if the Description cell is empty, it uses the description above (e.g. those Editor hotkeys)